rpcs3/Emu/RSX/VK/vkutils/device.cpp

```diff
@@ -234,7 +234,8 @@ namespace vk
 		float queue_priorities[1] = { 0.f };
 		pgpu = &pdev;
 
-		ensure(graphics_queue_idx == present_queue_idx || present_queue_idx == UINT32_MAX); // TODO
+		//ensure(graphics_queue_idx == present_queue_idx || present_queue_idx == UINT32_MAX); // TODO
```

Comment:
In case you're trying to "fix" the current crash, wasn't it line 255 that fails?

Reply:
Oh right, "fixed".